tools/alpine/detect-arch-var

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
```

Comment:
Can't this be moved into the Makefile, ie set a makefile variables for `uname -m` and then set other makefile variables based off that?